### PR TITLE
fixtures: added purging for redis and redisKvstore implementation;

### LINKED ISSUE #217: fixture-loader: purge redis
We already have #196 for mysql.

Now we want also purge redis.

### PR DESCRIPTION
**Pull request by applike-ss**
closes #217

### CODE REVIEW

File: pkg/fixtures/writer_redis.go [hunk 1]
```diff
@@ -36,26 +36,30 @@ type redisFixtureWriter struct {
...
 	return &redisFixtureWriter{
 		logger:    logger,
 		client:    client,
+		purger:    purger,
 		operation: *operation,
 	}
 }
 
 func (d *redisFixtureWriter) Purge() error {
-	return nil
```
Comment (bogdanfinn) [inline pkg/fixtures/writer_redis.go]: maybe the same logging here like in the other purger?

File: pkg/fixtures/writer_redis_kvstore.go [hunk 2]
```diff
@@ -11,33 +11,39 @@ import (
...
+func NewRedisKvStoreFixtureWriterWithInterfaces(logger mon.Logger, store kvstore.KvStore, purger *redisPurger) FixtureWriter {
 	return &redisKvStoreFixtureWriter{
 		logger: logger,
 		store:  store,
+		purger: purger,
 	}
 }
 
 func (d *redisKvStoreFixtureWriter) Purge() error {
-	return nil
```
Comment (bogdanfinn) [inline pkg/fixtures/writer_redis_kvstore.go]: maybe same logging here?

File: test/fixtures_redis_test.go [hunk 3]
```diff
@@ -74,26 +78,112 @@ func (s FixturesRedisSuite) TestRedis() {
...
+	assert.NoError(s.T(), err)
+
+	res, err = s.client.Get("gosoline-integration-test-test-application-kvstore-testModel-kvstore_entry_1").Result()
+
+	// should have created the item
+	assert.Error(s.T(), err)
+
+	res, err = s.client.Get("gosoline-integration-test-test-application-kvstore-testModel-kvstore_entry_2").Result()
+
+	// should have created the item
```
Comment (bogdanfinn) [inline test/fixtures_redis_test.go]: again a confusing comment?

Comment (bogdanfinn) [reply test/fixtures_redis_test.go]: sorry wrong line

File: test/fixtures_redis_test.go [hunk 4]
```diff
@@ -74,26 +78,112 @@ func (s FixturesRedisSuite) TestRedis() {
...
 	// should have created the item
 	assert.NoError(s.T(), err)
 	assert.JSONEq(s.T(), `{"name":"foo","age":123}`, res)
+
+	err = loader.Load(redisKvstoreEnabledPurgeFixtures())
+	assert.NoError(s.T(), err)
+
+	res, err = s.client.Get("gosoline-integration-test-test-application-kvstore-testModel-kvstore_entry_1").Result()
+
+	// should have created the item
```
Comment (bogdanfinn) [inline test/fixtures_redis_test.go]: again a confusing comment?

File: pkg/fixtures/writer_redis.go (hunk 1 above)
Comment (applike-ss) [reply pkg/fixtures/writer_redis.go]: it is already in the purger :+1:

**Pull request merged**